rct_image_tools/src/charuco_test.cpp

```diff
@@ -0,0 +1,60 @@
+#include "rct_image_tools/charuco_finder.h"
+
+int main(int argc, char** argv)
```

Comment:
Please convert this into a unit test instead of an executable per our discussion earlier